pkg/assessor/manifest/manifest.go

```diff
@@ -210,6 +227,38 @@ func useADDstatement(cmdSlices map[int][]string) bool {
 	return false
 }
 
+func sensitiveVars(cmd string) bool {
+	if !strings.Contains(cmd, "=") {
+		return false
+	}
+
+	for _, s := range sensitiveWords {
+		sensitiveWords = append(sensitiveWords, strings.ToUpper(s))
+	}
+	pat := strings.ReplaceAll(`.*(REP).*`, "REP", strings.Join(sensitiveWords, "|"))
+	r, _ := regexp.Compile(pat)
+
+	toklexer := shlex.NewLexer(strings.NewReader(strings.ReplaceAll(cmd, "#", "")))
+
+	for {
+		word, err := toklexer.Next()
+		if err == io.EOF {
+			break
+		}
+		if !strings.Contains(word, "=") {
+			continue
+		}
+		varName := strings.Split(word, "=")[0]
```

Comment:
You may catch a few false-positives with this approach, I can think of cases with command line arguments such as:

```
/bin/sh -c cmd --pass=$SECURE_PASSWORD
/bin/sh -c cmd --api_config=api.yaml
```